gcloud/datastore/client.py

```diff
@@ -441,8 +441,38 @@ def transaction(self):
 
     def query(self, **kwargs):
         """Proxy to :class:`gcloud.datastore.query.Query`.
-
```

Comment:
I just realized that's a really wierd docstring.

```python
"""Proxy to :class:`gcloud.datastore.query.Query`. Passes our ``project``."""
```
Should we just take out the "Passes our project" part?